src/main/java/hudson/plugins/timestamper/TimestamperOutputStream.java

```diff
@@ -90,10 +91,9 @@ public void write(byte[] b, int off, int len) throws IOException {
   }
 
   private void writeTimestamps(byte[] b, int off, int len) {
-    byte newlineCharacter = (byte) 0x0A;
     int lineStartCount = 0;
     for (int i = off; i < off + len; i++) {
-      if (previousCharacter == -1 || previousCharacter == newlineCharacter) {
+      if (previousCharacter == Integer.MIN_VALUE || previousCharacter == NEWLINE) {
```

Comment:
who adds the -1? Now we check MIN_VALUE but maybe -1 is the character used yet

Reply:
I don't understand your comment